numpy/lib/arraysetops.py

```diff
@@ -399,70 +400,74 @@ def in1d(ar1, ar2, assume_unique=False, invert=False):
     invert : bool, optional
         If True, the values in the returned array are inverted (that is,
         False where an element of `ar1` is in `ar2` and True otherwise).
-        Default is False. ``np.in1d(a, b, invert=True)`` is equivalent
-        to (but is faster than) ``np.invert(in1d(a, b))``.
+        Default is False. ``np.isin(a, b, invert=True)`` is equivalent
+        to (but is faster than) ``np.invert(isin(a, b))``.
 
         .. versionadded:: 1.8.0
 
     Returns
     -------
-    in1d : (M,) ndarray, bool
-        The values `ar1[in1d]` are in `ar2`.
+    isin : ndarray, bool
+        The values `ar1[isin]` are in `ar2`.
 
     See Also
     --------
+    in1d                  : The older, flattened version of this function
     numpy.lib.arraysetops : Module with a number of other functions for
                             performing set operations on arrays.
 
     Notes
     -----
-    `in1d` can be considered as an element-wise function version of the
-    python keyword `in`, for 1-D sequences. ``in1d(a, b)`` is roughly
-    equivalent to ``np.array([item in b for item in a])``.
+    `isin` can be considered as an element-wise function version of the
+    python keyword `in`. ``in1d(ar1, ar2)`` is roughly equivalent to
+    ``in1d = np.vectorize(lambda x: x in ar2); in1d(ar1)``.
```

Comment:
I would keep the original example, which I think is a little clearer than using `np.vectorize`.